tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Shape.cpp

```diff
@@ -146,26 +147,32 @@ std::vector<std::vector<std::complex<double>>> generate_random_coefs(
     size_t l_max, size_t m_max, gsl::not_null<std::mt19937*> generator) {
   std::vector<std::vector<std::complex<double>>> coefs{};
   // if the coefficients are made too large, the map has a good chance of
-  // mapping through the center, causing the test to fail
-  std::uniform_real_distribution<double> coef_dis{0.1, 0.2};
-  const auto ran = [&coef_dis, &generator]() { return coef_dis(*generator); };
+  // mapping through the center, causing the test to fail which is why we damp
+  // higher mode coefficients. The factor was found empirically by trial and
+  // error.
+  std::uniform_real_distribution<double> coef_dist{0., 1.};
 
   for (size_t l = 0; l <= l_max; ++l) {
-    std::vector<std::complex<double>> tmp{ran()};
+    // m=0 is real
+    std::vector<std::complex<double>> tmp{
+        make_with_random_values<double>(generator, coef_dist, 1) /
```

Comment:
if you just need one random double, do `coef_dist(*generator)` as you did before. My suggestion was to use `make_with_value` to generate the full vector of complex numbers (then remove the imag part of m=0), though I'm not sure if it works.

Reply:
I did not get this to work. I dont think you can `make_with_value` a `std::vector` at the moment.

Reply:
ok, then pick your choice of generating these random values :)